docs/operate/get-started/supported-hardware.md

```diff
@@ -5,5 +5,367 @@ weight: 20
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Use supported hardware with your machine."
+modulescript: true
 ---
+
+Viam supports a wide variety of sensors, cameras, motors, robotic arms, and other physical hardware.
+Viam also supports various software services such as [data capture](/data-ai/get-started/capture-sync/), [computer vision](/data-ai/ai/create-dataset/), and [motion planning](/operate/mobility/move-arm/), designed to integrate seamlessly with the hardware driver modules.
+
+Any hardware that is not already supported by a Viam {{< glossary_tooltip term_id="module" text="module" >}} can be added into Viam's system of modular resources by [creating a new module](../other-hardware/) that provides a driver for the hardware.
+You can also create a module to support new software-only functionality.
+
+In addition to physical hardware, there are "virtual" hardware modules that do not directly drive any physical hardware, but rather augment physical hardware with another layer of abstraction, or add functionality that has little to do with hardware, for example:
+
+- [A "camera" that takes a camera feed from a physical camera, and crops it, overlays it, or otherwise transforms the output](/components/camera/transform/)
+- [A "sensor" that allows you to designate a primary sensor and backup sensors in case of failure](https://github.com/viam-modules/failover)
+- [A ChatGPT integration module](https://github.com/jeremyrhyde/chat-gpt-module)
+- [A "sensor" that returns all active I2C addresses as sensor readings](https://github.com/michaellee1019/i2cdetect)
+
+These software-only "hardware" modules implement the same [component APIs](/dev/reference/apis/#component-apis) as physical hardware modules.
+
+{{% alert title="In this page" color="tip" %}}
+
+- [What hardware is already supported?](#supported-hardware)
+- [Add supported hardware to my machine's configuration](#configure-hardware-on-your-machine)
+
+{{% /alert %}}
+
+{{% alert title="See also" color="tip" %}}
+
+- [Support other hardware by creating a new module](../other-hardware/)
+
+{{% /alert %}}
```

Comment:
I think you may want to remove these. You may also want to postpone explaining virtual hardware till later on the page. It currently takes too long to get to the part where you do things

Reply:
Put it into a little details expander-LMK what you think of that.